packages/components/badge/src/badge.stories.ts

```diff
@@ -0,0 +1,112 @@
+import type { BadgeSize, BadgeVariant } from './badge.js';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { Icon } from '@sl-design-system/icon';
+import { faCheck, faGear } from '@fortawesome/pro-regular-svg-icons';
+import { html, nothing } from 'lit';
+import '../register.js';
+
+interface Props {
```

Comment:
You can do `type Props = Pick<Badge, 'size' | 'variant'> & { icon?: boolean; text?: string };`